rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -79,70 +96,94 @@ class TestStepSelector {
 
 				})
 		]
-		val page = PlatformUI.workbench.activeWorkbenchWindow.activePage;
-		page.addPartListener(new IPartListener() {
-
-			override partActivated(IWorkbenchPart part) {
-				if (part instanceof ITextEditor) {
-					refreshView(null)
+		
+		Job.getJobManager().addJobChangeListener(new JobChangeAdapter() {
```

Comment:
I'm not sure about this one. I think this will work to fill the view initially, but after that it will trigger an update on the `TestSteptSelector` every time the workspace is build. So it will also trigger an update if we just change any Java file or and TCL file.